Explorer/Assets/DCL/InWorldCamera/PassportBridge/IPassportBridge.cs

```diff
@@ -0,0 +1,9 @@
+using MVC;
+
+namespace DCL.InWorldCamera.PassportBridge
+{
+    public interface IPassportBridge
```

Comment:
@fcolarich does this look familiar? 😉

Reply:
LETS MAKE MORE BRIDGES

Reply:
until we have a class IBridgeBridgeBridge